src/transaction_builder.js

```diff
@@ -206,15 +210,22 @@ export class TransactionBuilder {
       this.timebounds.maxTime = this.timebounds.maxTime.getTime() / 1000;
     }
 
-    this.timebounds.minTime = UnsignedHyper.fromString(this.timebounds.minTime.toString());
-    this.timebounds.maxTime = UnsignedHyper.fromString(this.timebounds.maxTime.toString())
-    
+    this.timebounds.minTime = UnsignedHyper.fromString(
+      this.timebounds.minTime.toString()
+    );
+    this.timebounds.maxTime = UnsignedHyper.fromString(
+      this.timebounds.maxTime.toString()
+    );
+
     attrs.timeBounds = new xdr.TimeBounds(this.timebounds);
 
     const xtx = new xdr.Transaction(attrs);
     xtx.operations(this.operations);
 
-    const xenv = new xdr.TransactionEnvelope({ tx: xtx });
+    const xenv = new xdr.TransactionEnvelope.envelopeTypeTx(
+      new xdr.TransactionV1Envelope({ tx: xtx })
```

Comment:
shouldn't we be creating v0 transactions?

Reply:
@tamirms yeah, turns out I did that but forgot to push that commit. It is now fixed.